ports/gonk/Cargo.lock

```diff
@@ -1310,6 +1310,7 @@ dependencies = [
  "fnv 1.0.0 (registry+https://github.com/rust-lang/crates.io-index)",
  "html5ever 0.2.6 (registry+https://github.com/rust-lang/crates.io-index)",
  "hyper 0.6.15 (registry+https://github.com/rust-lang/crates.io-index)",
+ "image 0.4.0 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
a new version of image was recently pushed (0.4.1) that might be worth including in this so that Google News bug can get closed